### DISCUSSION TITLE
Issue with API Authentication After Latest Update

### DISCUSSION THREAD

**Discussion by hazama0x** [hazama0x/rakoko-app#76] [Q&A] [answered]
Hi team,

I'm encountering an issue with the API authentication in my project after the recent update. Previously, everything worked fine, but after upgrading to the latest version of the package, I keep receiving 401 Unauthorized errors when making requests to the API.

Here are the steps I followed:

1. Updated the API package to the latest version (v2.3.0).
2. Used the new authentication headers as per the updated documentation.
3. Verified my credentials and API key, which are correct.

Despite this, I still can't authenticate properly. Has anyone else faced a similar issue or found a solution?

`{
  "error": "Unauthorized",
  "message": "Invalid API key provided"
}`

Looking forward to any suggestions or guidance on resolving this.

Thanks!

**Comment by ghost** [comment, marked as answer]
Hi there!

I faced a similar issue when updating to the same version. Here's what worked for me:

1. **Double-check the header structure**: Ensure you're using the correct header format. The latest update requires you to use Authorization: Bearer <API_KEY> instead of the older X-API-Key format.
2. **Clear your local cache**: Sometimes, after an update, old session data can cause issues. Try clearing your cache and retrying the request.
3. **Re-generate API key**: If the above steps don't work, it might help to regenerate a new API key from the dashboard and test with that.

Here's an example of the correct request structure:

`
curl -X GET "https://api.example.com/data" \
-H "Authorization: Bearer YOUR_API_KEY" \
-H "Content-Type: application/json"
`

Let me know if this resolves the issue or if you need further assistance!
